Project/core/src/main/java/View/screen/SetupScreen.java

```diff
@@ -46,12 +46,12 @@ private void setupUI() {
 
         for (int i = 2; i <= 5; i++) {
             final int value = i;
-            TextButton playersButton = new TextButton(i + " players", application.skin);
-            root.add(playersButton).pad(20);
-            playersButton.addListener(new ChangeListener() {
+            TextButton usersButton = new TextButton(i + " users", application.skin);
```

Comment:
Same here, I think "players" sound better and shouldn't change just because or variable names did. 
```suggestion
            TextButton usersButton = new TextButton(i + " players", application.skin);
```